auth/src/main/java/com/firebase/ui/auth/util/signincontainer/SignInDelegate.java

```diff
@@ -248,8 +248,12 @@ private void startAuthMethodChoice() {
                     break;
                 case PhoneAuthProvider.PROVIDER_ID:
                     // Go directly to phone flow
+                    String phone = idpConfigs
+                            .get(0)
```

Comment:
This same logic is used above so we should probably extract a `firstIdpConfig` and replace `firstProvider` with the method call.

Reply:
Good point, done.